webpack.prod.config.js

```diff
@@ -157,7 +155,7 @@ module.exports = {
       // comments: true,//debug
 
       beautify: false,//prod
-      // disable mangling because of a bug in angular2 beta.1, beta.2, and beta.3
+      // disable mangling because of a bug in angular2 beta.1, beta.2 and beta.3
```

Comment:
As a person who's mother tongue is German, the comma before the "and" somehow doesn't look right. ;-) However, I didn't know that it was actually optional from a grammatical point of view, so sorry about correcting it.